src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,113 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
 import './FindMovie.scss';
 
-import { MovieCard } from '../MovieCard';
 import movies from '../../api/movies.json';
+import { getMovie } from '../../api/api';
 
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { MovieCard } from '../MovieCard';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ addMovie }) => {
+  const [movie, setMovie] = useState(movies[0]);
+  const [error, setError] = useState(false);
+  const [inputValue, setInputValue] = useState('');
+
+  const findMovie = async() => {
+    const result = await getMovie(inputValue);
+
+    if (result.Response === 'False') {
+      setError(true);
+
+      return;
+    }
+
+    setMovie({
+      title: result.Title,
+      description: result.Plot,
+      imgUrl: result.Poster,
+      imdbUrl: `https://www.imdb.com/title/${result.imdbId}`,
```

Comment:
movie's imdbUrl links are incorrect (404 error), maybe because the prop name is imdbID, not imdbId or you forgot to put a slash in the end 
>  `https://www.imdb.com/title/${movie.imdbID}/`

Reply:
Исправила, спасибо)

Reply:
fixed, thx)